Header header
float32[] x
float32[] y
float32[] z
int32[] frames
int32[] array_sizes
int32[] num_clusters
float32 scan_time
